Dockerfile.in

```diff
@@ -1,5 +1,5 @@
 #@follow_tag(registry.redhat.io/ubi9:latest)
-FROM registry.redhat.io/ubi9/ubi:9.1.0-1817 AS builder
+FROM registry.redhat.io/ubi9:9.2-755.1697625012 AS builder
```

Comment:
If  you do not need this in our version of upstream CI, I would remove the file entirely given we do not sink them to the downstream

Reply:
I'd keep them around to be able to test the upstream as if it was downstream.

Reply:
We can not fully mimic cpaas because of the way cachito works and it pulls in deps. The original intent was to figure a way to sync but it really became something that is just not possible.  The file isn't needed in the upstream